#Empty service to trigger the findCenter routine. It will also set the position to zero.
---